app/models/poem.rb

```diff
@@ -1,6 +1,18 @@
+# frozen_string_literal: true
+
 class Poem < ApplicationRecord
+  POEM_STATUSES = [
```

Comment:
Might be better to call this `STATUSES`? It will always be prefixed by the `Poem` namespace outside of this file so it will read well.